### DISCUSSION TITLE
Add to m1/2/3/4 macOS App Store?

### DISCUSSION THREAD

**Discussion by yihuanlin** [infinitepower18/Homepage-MobileSafari#14] [General]
Hello there,

Would the app be on mac app store for m1/2/3/4 macs? I know that safari on mac can already change homepage but I want to do this via a script. On iOS I can set a shortcut to change homepage url of this extension. If this app is on mac app store I can tell apple script to run the same shortcut in scripting. Or is there a way to change the homepage on shell?

Thank you.

**Comment by infinitepower18** [comment]
There is a shell script AFAIK but I have just enabled the app to be installed on Mac in case you find using the shortcut easier.
Are you now able to achieve what you want to do?

**Reply by yihuanlin** [reply]
Thank you! Yes it works. Btw is the shell script `sudo defaults write com.apple.Safari HomePage -string "https://example.com"`? It does not seem to work on the latest version of macOS.

**Reply by infinitepower18** [reply]
I haven't tried it but I read that it's not reliable.

I haven't found any issues when running the Homepage extension on my MacBook Pro with macOS 15.2 so I'll leave it available on the Mac App Store.
